Expensify/App
# Remove unnecessary library for callback methods

This was only used in one spot and it was over-engineered because some options and methods were never used 
### Fixed Issues
None

### Tests & QA
1. Go offline
2. Send a message in a chat and verify it shows as pending
3. Go back online
4. Verify the message is sent and no longer shows as pending
